data_entry/views.py

```diff
@@ -70,24 +156,57 @@ def collectData(request):
         context = {}
         return render(request, 'data_entry/collectdata.html', context)
 
+"""
+    View Data Page
+
+    if a POST request is received:
+        An entry is being deleted
+        TODO: A search has been made
+    otherwise load database entries
+
+"""
 def viewData(request):
-    #when they want to do filters
     if request.method == 'POST':
         method = request.POST['method']
+        # delete entry from database
         if method == "delete":
             entry_id = request.POST['entry_id']
             Test.objects.filter(id=entry_id).delete()
             entries = Test.objects.all()
             context = {'entries': entries}
             return render(request, 'data_entry/viewdata.html', context)
         
+        # filter entries
```

Comment:
@lealal Is this supposed to be commented out?

Reply:
Yes, I doubt that code will work. It is the outline of how to filter entries based on a search parameter. I think Johnny was going to work on that, so he can use this as a guideline